tests/jest/unit/modules/default-values/assets/js/editor/handlers/global-values.test.js

```diff
@@ -0,0 +1,103 @@
+import GlobalValues from 'elementor/modules/default-values/assets/js/editor/handlers/global-values';
+
+describe( 'Default Values - Handlers - Global Values', () => {
+	test( 'appendSettingsForSave', () => {
+		// Arrange
+		const globalValues = new GlobalValues( {
+			test_widget: {
+				controls: {
+					control_default_color: {
+						global: {
+							default: 'globals/colors?id=sec',
+						},
+					},
+					control_default_typography: {
+						global: {
+							default: 'globals/typography?id=sec',
+						},
+					},
+				},
+			},
+		} );
+
+		const container = {
+			model: {
+				attributes: {
+					widgetType: 'test_widget',
+				},
+			},
+			settings: {
+				attributes: {
+					__globals__: {
+						title_color: 'globals/colors?id=secondary',
+						typography_typography: 'globals/typography?id=primary',
+						control_default_color: 'globals/colors?id=primary',
+						local_settings: 'global index',
+						empty_global: '',
+					},
+				},
+			},
+		};
+
+		// Act
+		const result = globalValues.appendSettingsForSave( {
+			local_settings: 'red',
+			another_local_settings: 'blue',
```

Comment:
```suggestion
			local_setting: 'red',
			another_local_setting: 'blue',
```

Without 's', also in other places.

Reply:
Done